stalkbook/views.py

```diff
@@ -1,8 +1,44 @@
+import flickrapi
+import json
+import re
+import webbrowser
+
 from django.shortcuts import render
+from django.http import HttpResponse
 from django.conf import settings
-from django.contrib.auth.decorators import login_required
 
-# Create your views here.
-#@login_required(login_url="login/")
-def home(request):
-    return render(request, 'main.html', {'STATIC_URL': settings.STATIC_URL})
+
+from django.shortcuts import render
+from django.template import Context, loader
+
+
+def home(req):
+    return HttpResponse("hiii")
+
+# It returns all the urls for photos with the given tag
+
+
+def flickr_extract(request):
```

Comment:
There should be a view for `GET` request. And this view should call a function `get_flickr_images` to get the image urls. And in future when we add some more integrations, lets say `instagram` then we will create a function like `get_instagram_images`. And aggregate the results from both and return it in the response.